caliban_toolbox/utils/io_utils.py

```diff
@@ -200,13 +200,16 @@ def load_npzs(crop_dir, log_data, verbose=True):
             if os.path.exists(npz_path):
                 temp_npz = np.load(npz_path)
 
+                # determine how labels were named
+                labels_key = 'y' if 'y' in list(temp_npz.keys()) else 'annotated'
```

Comment:
I think you can probably just use `'y' in temp_npz`.  Checking if a value is in a list is O(N) whereas checking if a key is in a dictionary is O(1). I'm not sure, but I'm guessing the `in` operator behaves dictionary-like with these files.

Reply:
That's pretty cool!